<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSurface</class>
 <widget class="QDialog" name="EditSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>57</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Surface</string>
  </property>
  <widget class="QPushButton" name="AddTriangleB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Triangle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteTriangleB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Triangle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SwapEdgeB">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Swap Edge</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SelectSurfaceCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Surface</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
